Solutions/CrowdStrike Falcon Endpoint Protection/Package/mainTemplate.json

```diff
@@ -870,7 +870,7 @@
                 "eTag": "*",
                 "displayName": "CrowdstrikeFalconEventStream",
                 "category": "Samples",
-                "functionAlias": "CrowdStrikeFalconEventStream",
+                "functionAlias": "CrowdstrikeFalconEventStream",
```

Comment:
Revert this change, it will create problem while executing all the analytical and hunting queries **CrowdStrikeFalconEventStream**